anvil_schema/fss_index_data_dictionary/fss_index_data_dictionary.json

```diff
@@ -0,0 +1,888 @@
+      "intPartitionOptions": null,
+      "reqiured": false
+    },
+    {
+      "name": "anvil_diagnosis",
+      "description": "Contains information about conditions or diagnoses related to the donor of a biosample.",
+      "columns": [
+        {
+          "name": "diagnosis_id", 
+          "description": "A unique identifier property for a disease or condition has been reported in this entity.",
+          "datatype": "string", 
+          "array_of": false, 
+          "required": true
+        },
+        {"name": "donor_id", 
+        "description": "A unique identifier for the donor associated with this diagnosis or condition.",
+        "datatype": "string", 
+        "array_of": false, 
+        "required": false
+      },
+        {"name": "disease", 
+        "description": "A human-readable property that identifies a disease or condition has been reported in this entity.",
+        "datatype": "string", 
+        "array_of": true, 
+        "required": false
+      },
+        {
+          "name": "diagnosis_age_unit", 
+          "description": "Unit of time measurement the age at diagnosis is documented (for example, days, years, etc.)",
+          "datatype": "string", 
+          "array_of": false, 
+          "required": false
+        },
+        {
+          "name": "diagnosis_age_lower_bound", 
+          "description": "Lower bound for age of donor at time diagnosis was made. If exact age is known, this should be the same as the upper bound.",
+          "datatype": "float", 
+          "array_of": false, 
+          "required": false
+        },
+        {
+          "name": "diagnosis_age_upper_bound", 
+          "description": "Upper bound for age of donor at time diagnosis was made. If exact age is known, this should be the same as the lower bound.",
+          "datatype": "float", 
+          "array_of": false, 
+          "required": false
+        },
+        {
+          "name": "onset_age_unit", 
+          "description": "Unit of time measurement the age at which onset of disease is documented (for example, days, years, etc.)",
+          "datatype": "string", 
+          "array_of": false, 
+          "required": false
+        },
+        {
+          "name": "onset_age_lower_bound", 
+          "description": "Lower bound for age of donor at documented time of onset. If exact age is known, this should be the same as the upper bound.",
+          "datatype": "float", 
+          "array_of": false, 
+          "required": false
+        },
+        {
+          "name": "onset_age_upper_bound", 
+          "description": "Upper bound for age of donor at documented time of onset. If exact age is known, this should be the same as the lower bound.",
+          "datatype": "float", 
+          "array_of": false, 
+          "required": false
+        },
+        {
+          "name": "phenotype", 
+          /// no idea what to put here wrt how this applies to the table
+          "description": "",
```

Comment:
A human-readable property that describes a disease, condition, or phenotype that has been reported in this entity in a human readable manner (as opposed to a code/id/etc). If multiple values exist, they must be synonymous.